### ISSUE TITLE
I did a swap from rUSDT to DOC but appear as rUSDT to rBTC

### ISSUE THREAD

**Issue by bitcoinuser** [DistributedCollective/Sovryn-frontend#1117]
Hi,

I did a swap from rUSDT to DOC but appear as rUSDT to rBTC. (https://explorer.rsk.co/tx/0x3305b66dcf76e9d3f58fbc53588e01b558b266c1d8191787e150f9780a6f5e59)

![Captura de tela de 2021-05-17 11-32-19](https://user-images.githubusercontent.com/1178856/118506577-d1592f80-b703-11eb-8d05-c06241110161.png)

**Comment by soulBit** [comment]
I've tested this and can replicate when swapping between any pair that uses rBTC as intermediate step (e.g. USDT->SOV, USDT->BPRO, USDT->DoC, DoC->USDT):

![image](https://user-images.githubusercontent.com/765825/119159761-79962d80-ba4f-11eb-83cc-68b26e5db333.png)

It appears that the call to backend api endpoint is not returning the correct "to" token, and just returning rBTC instead - e.g:
https://testnet.sovryn.app/backend/events/conversion-swap/0xE8789B348c978B19DfB915A10b761069915C237b

Sample data from backend call for USDT -> DoC swap:
```
{
    "beneficiary": "0xE8789B348c978B19DfB915A10b761069915C237b",
    "from_token": "0x4d5A316d23EBe168D8f887b4447BF8DBfA4901cc", //USDT
    "to_token": "0x69FE5cEC81D5eF92600c1A0dB1F11986AB3758Ab", //RBTC
    "timestamp": 1621606889,
    "transaction_hash": "0x75c565e9d78030e94bb8ae34ac57fec7cc52988c06b4abe1aea0777e11d383d9",
    "returnVal": {
        "_trader": "0xE8789B348c978B19DfB915A10b761069915C237b",
        "_fromToken": "0x4d5A316d23EBe168D8f887b4447BF8DBfA4901cc", //USDT
        "_toToken": "0x69FE5cEC81D5eF92600c1A0dB1F11986AB3758Ab", //RBTC
        "_fromAmount": "50000000000000000000",
        "_toAmount": "1287840000000000"
    }
}
```
The token transfers are performed correctly and appear in block explorer for these transactions, so I believe the issue is in api endpoint - my tests are:
- USDT -> SOV https://explorer.testnet.rsk.co/tx/0x3f02712622c0624ade614ae3140b265051c02d0836e6e0520651a839f9ed94a6?__ctab=Token%20Transfers
- USDT -> BPRO https://explorer.testnet.rsk.co/tx/0x6f87ff05e1957fdc1db15a672f593a43676beb503cf2d67c6844ece99c1bfac7?__ctab=Token%20Transfers
- DoC -> USDT https://explorer.testnet.rsk.co/tx/0xf68020466102bbaa4aa99cbf98eb72396ebd61488e60eb6b8e6e1ddb17b96dc7?__ctab=Token%20Transfers
- USDT -> DoC https://explorer.testnet.rsk.co/tx/0x75c565e9d78030e94bb8ae34ac57fec7cc52988c06b4abe1aea0777e11d383d9?__ctab=Token%20Transfers

**Comment by bitcoinuser** [comment]
On build ID: d21a263, now a new line appeared. So looks like the swap was from rUSDT to rBTC then rBTC to DoC. But I still think should show just only one line: rUSDT to DoC. 

![Captura de tela de 2021-05-31 14-13-18](https://user-images.githubusercontent.com/1178856/120224881-a5b56980-c21a-11eb-92ec-9d57cd8160ac.png)

**Comment by bitcoinuser** [comment]
Fixed on Build ID: 4889804
